wdl2cwl/main.py

```diff
@@ -60,10 +61,66 @@ def load_wdl_objects(
             return self.load_wdl_task(obj)
         raise Exception(f"Unimplemented type: {type(obj)}: {obj}")
 
+    def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
+        """Load WDL workflow and convert to CWL."""
+        inputs: List[cwl.WorkflowInputParameter] = []
```

Comment:
For getting the workflow inputs, have you considered looking at `obj.available_inputs`? 
https://miniwdl.readthedocs.io/en/latest/WDL.html#WDL.Tree.Workflow.available_inputs

That will probably work better for workflows with many layers of nesting and imports as MiniWDL will figure out all of that for you
``` python
for wf_input in obj.available_inputs:
  info: Any = wf_input.info  # Always None?
  name: str = wf_input.name   # like `align_and_count.reads_bam`
  value: WDL.Tree.Decl = wf_input.value  # like WDL.Tree.Decl(type=WDL.Type.File, name="reads_bam")
```

Reply:
Yeah. I saw this and the required_inputs to and it looks interesting. Maybe we can explore it with another PR. Although I observed that the available_inputs don't carry the information of the namespace they are from. Which seems important for determining the ID of the input. So in the case where the import is more than one, we can't effectively return the right namespace for the right available_input.

I will look out for this when exploring files with more than one imports and see how the available inputs can be used.

Reply:
I think ` required_inputs ` is more for those running a WDL workflow, so we shouldn't look at it.

Reply:
> Maybe we can explore it with another PR.

Can you make an issue then?

Do that and resolve @kinow 's comment and I'll merge this, thanks!